python/ngraph/ops.py

```diff
@@ -527,6 +527,24 @@ def convert(node, new_type, name=None):  # type: (Node, NumericType, str) -> Nod
     return Convert(node, new_element_type)
 
 
+@nameable_op
+def gelu(node, name=None):  # type: (NodeInput, str) -> Node
+    r"""Perform Gaussian Error Linear Unit operation element-wise on data from input node.
```

Comment:
Was this 'r' character intended?

Reply:
Yes, it was. `r` is a prefix for Python raw string. It treats backslash as a literal character, not as an escape character.